cetloxblock/CetInsights/CetInsightsPage.html

```diff
@@ -0,0 +1,83 @@
+﻿<!--Add the following HTML code to CET Insights page's RAW html-->
+<script src="https://login.cet.ac.il/Scripts/login.js"></script>
+<script src="//ebag.dev.cet.ac.il/CetSso.js"></script>

```

Comment:
Cool relative protocol :) But as the notice [in this article](https://www.paulirish.com/2010/the-protocol-relative-url/) says, it's probably good to be explicit about the protocol, i.e. `https://` rather than `//`.

Reply:
Also, I tried loading http://ebag.dev.cet.ac.il/CetSso.js or https://ebag.dev.cet.ac.il/CetSso.js, and it doesn't work for me.

Reply:
This whole file was not supposed to be a part of the component and will be deleted.